src/views/Events/index.js

```diff
@@ -123,8 +123,9 @@ export default class Events extends Component {
          * So, when includePast is true, we set it to false so that togglePastEvents() will set it
          * back to true
          */
-        this.setState({ includePast: false });
-        this.togglePastEvents();
+        this.setState({ includePast: false }, () => {
+          this.togglePastEvents();
```

Comment:
This code is OK, but it's creating a new unnamed function that just calls a named function.  It would be slightly simpler and cleaner to say
  this.setState({ includePast: false }, this.togglePastEvents);